### PR TITLE
Greedy Algorithm (Activity Selection) in JAVA with link attached in Readme file

### PR DESCRIPTION
**Pull request by shobhu98**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by shobhu98** [comment]
Waiting for the maintainers to merge the request

**Review by siddharth25pandey — APPROVED** [review]
@shobhu98 Looks fine to me
Able to merge.
Thanks!

File: Java/Greedy_Algorithms/Active_Selection.java
Comment (Kajol-Kumari) [inline Java/Greedy_Algorithms/Active_Selection.java]: please prove one extra blank line at the end of the file.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@shobhu98 I can see a lot of empty white spaces in the code. Please use some code formatting tool to format your code. Rest looks good!

File: Java/Greedy_Algorithms/Active_Selection.java
Comment (shobhu98) [reply Java/Greedy_Algorithms/Active_Selection.java]: Done. I removed those extra spaces .

**Comment by shobhu98** [comment]
I have removed the spaces I guess you are good to go.

**Review by SKAUL05 — APPROVED** [review]

**Comment by shobhu98** [comment]
Waiting for the maintainers to merge the request.

**Comment by shobhu98** [comment]
It is now more than 10 days. I have done some more work waiting dor this request to merge first

**Pull request merged**
